packages/node/src/events/propose-install-virtual/operation.ts

```diff
@@ -19,15 +19,8 @@ export async function setAppInstanceIDForProposeInstallVirtual(
   appInstanceId: string,
   incomingAddress: Address
 ) {
-  const channel = await getChannelFromPeerAddress(
-    selfAddress,
-    incomingAddress,
-    store
+  await store.addAppInstanceProposal(
```

Comment:
I'm generally not a fan of doing this because _unless you look at the function signature_ to see what it's expecting (which is too tedious when you're just _looking_ at its invocation), these inliners are unnecessary optimizations that take away from easy legibility.

In the previous case, it's extremely obvious immediately what's being passed in.

Reply:
I don't understand to be honest, this change only removes the variable names but is otherwise the same thing. I think you can infer that the parameter being passed in is a "channel" if the function name is `getChannelFromPeerAddress` and the second parameter is a "proposedAppInstance" if the value is `new ProposedAppInstanceInfo`

Reply:
In this case you can infer, but it might not always be super clear.

And because it's not always clear, then you'd end up with sometimes having this style and sometimes having that style, thus introducing stylistic inconsistencies.

So the explicit variable naming is consistently erring on the side of easier legibility

Reply:
Yeah I'd be in favor of that

That'd also remove parameter ordering concerns